common/db.go

```diff
@@ -223,6 +265,45 @@ func (db *DB) LastSeenBlock() (*big.Int, error) {
 	return big.NewInt(lastSeenBlock), nil
 }
 
+func (db *DB) UpdateOrchs(orch *DBOrch) error {
```

Comment:
This function (and the associated prepared statement) only updates a single orchestrator; perhaps it should be named `UpdateOrch` instead?